src/main/java/net/openhft/chronicle/wire/internal/MissingClassOrClassAliasException.java

```diff
@@ -0,0 +1,16 @@
+package net.openhft.chronicle.wire.internal;
+
+public class MissingClassOrClassAliasException extends RuntimeException {
```

Comment:
If we DO want a missing class or alias to break deserialization then we should have a consistent error to represent that. It is represented by NoClassDefFoundError/ClassNotFoundRuntimeException/IllegalArgumentException depending on where it's thrown.